modules/backend/hooks/usePrefetchOnAppStart.tsx

```diff
@@ -24,42 +20,20 @@ export const usePrefetchOnAppStart = () => {
       return
     }
 
-    /* eslint-disable @typescript-eslint/no-explicit-any */
     try {
       /* Define and prefetch standard queries */
-      const queries = [
-        notificationSettingsOptions(),
-        visitorCardsOptions(),
-        announcementsOptions(locale),
-      ]
-      await Promise.all(
-        queries.map((query) => queryClient.prefetchQuery<any, any, any, any>(query)),
-      )
+      const queries = [visitorCardsOptions(), announcementsOptions(locale)]
 
-      /* Define and prefetch infinite queries (must be prefetched separately by `prefetchInfiniteQuery`) */
-      const verifiedEmails = verifiedEmailsInfiniteOptions()
-      const infiniteQueries = [verifiedEmails]
+      // refresh verified emails before prefetching visitor cards
+      await clientApi.verifiedEmailsControllerRefreshVerifiedEmail()
```

Comment:
I'm just thinking about placement of this call, it's little bit misleading to find it here in `usePrefetch...` hook. But I don't have a suggestion, so it's okay to leave it here, because it's related to verifiedCards prefetch.

Reply:
it's post request, not get, so it cannot be added to queries, but it needs to be called and I think best place for it is here